docs/sources/tempo/getting-started/docker-example.md

```diff
@@ -44,7 +44,7 @@ To follow this guide, you need:
 
 1. Start the services defined in the docker-compose file:
    ```bash
-   docker compose up -d
+   docker-compose up -d
```

Comment:
This fix had me digging into the docker compose commands. Both `docker compose` and `docker-compose` versions work. However, docker-compose is a v1 implementation that is being deprecated in favor of docker compose (see [this thread](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose)). 

I did not know this. I'll update the docs. Thank you!

```suggestion
   docker compose up -d
```

Reply:
Okay, wow, for me `docker compose` doesn't work at all, i installed docker and docker-compose through brew in mac. 

I get an error saying `docker: 'compose' is not a docker command.`, docker version 24.0.7.

Seems like the space version is only included in the docker for desktop system for now, and that linux and mac users installing through cli have to do extra steps in order for it to work.

I learned someething new today as well, thank you :)